payments-core/src/test/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherConfirmationActivityTest.kt

```diff
@@ -120,52 +116,6 @@ class PaymentLauncherConfirmationActivityTest {
         }
     }
 
-    @Test
```

Comment:
the tests are moved to `PaymentLauncherViewModelTest` to match [similar tests](https://github.com/stripe/stripe-android/blob/b1f1158887f342b059470dab934695e1196af6d9/paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt#L732)